.gitignore

```
@@ -5,3 +5,5 @@
 /var/
 /vendor/
 ###< symfony/framework-bundle ###
+
+.php_cs.cache
```

```suggestion
*.cache
```

If we use this pattern, then we don’t have to change this file again when we upgrade php-cs-fixer to v3.